examples/universal-cnf/vppagent/pkg/config/config_endpoints.go

```diff
@@ -62,7 +68,12 @@ func NewProcessEndpoints(backend UniversalCNFBackend, endpoints []*nseconfig.End
 			IPAddress:              e.VL3.IPAM.DefaultPrefixPool,
 			Routes:                 nil,
 		}
-
+		var err error
+		service := newService(e.VL3.IPAM.ServerAddress)
```

Comment:
Agree with Mihai on the var name.  Also, though, this need to be able to function if there is no server address defined so we don't want to call the constructor/IpamServiceFactory if the IPAM server address hasn't been configured.

Reply:
done